test/replication-luatest/gh_7932_recovery_mixed_transactions_test.lua

```diff
@@ -0,0 +1,163 @@
+local t = require('luatest')
+local cluster = require('luatest.replica_set')
+local fio = require('fio')
+
+local g = t.group('gh_7932')
+
+g.before_each(function(cg)
+    cg.cluster = cluster:new({})
+    cg.server = cg.cluster:build_and_add_server{
+        alias = 'server',
+        box_cfg = {
+            instance_uuid = '1a685e63-24cd-4fc5-9532-bf85f649e0ab',
+        },
+    }
+    cg.cluster:start()
+end)
+
+g.after_each(function(cg)
+    cg.cluster:drop()
+end)
+
+g.test_good_xlog_with_force_recovery = function(cg)
+    cg.server:exec(function()
+        box.schema.space.create("test", {id = 512})
+        box.space.test:create_index("primary")
+    end)
+
+    t.assert_equals(cg.server:exec(function()
+        return box.space.test.id
+    end), 512)
```

Comment:
You do not need to create the space in the tests, because all your test xlogs already contain space creation.

Reply:
Done